src/commands/flow/ReleaseNumber.ts

```diff
@@ -56,6 +69,7 @@ export class ReleaseNumber {
     }
 
     public toString(): string {
-        return this.master ? ' master' : `${this.major}.${this.minor}.${this.patch}`;
+        return this.defaultBranch ? 'default' : `${this.major}.${this.minor}.${this.patch}`;
```

Comment:
Is this method used at all?

Reply:
I think it is being used in Upmerge.ts / line 118:
`                    console.log('is in release:', releaseNumber);`